src/components/ProjectMetaDataTable/ProjectDetailsComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import Chat from '@/components/ProjectChatBox/Chat';
+import { ProjectCollaborators } from '../ProjectCollaborators';
+import ProjectDetailsCard from '../ProjectDetailsCard';
+import { ProjectDetailsComponentProps } from '@/types/const';
+
+const ProjectDetailsComponent = ({ project }: ProjectDetailsComponentProps) => {
+  if (!project) {
+    return null;
+  }
```

Comment:
avoid returning `null` in a react component
- the `project` prop is already known to be required in the parent. Update the parent component to conditionally render  `<ProjectDetailsComponent />` only if `project` resolves to `true`.